src/lib/flags.ts

```diff
@@ -8,9 +8,7 @@ const defaultFeatureFlag = ({ inventory }: GameState) =>
 const testnetFeatureFlag = () => CONFIG.NETWORK === "amoy";
 
 const clashOfFactionsFeatureFlag = () => {
-  if (testnetFeatureFlag()) return true;
-
-  return Date.now() > new Date("2024-05-01T00:00:00Z").getTime();
+  return true;
```

Comment:
We should actually just remove these old feature flags.

Reply:
I was going to do that in a separate PR.